Packs/Vectra_AI/Integrations/VectraDetect/VectraDetect.yml

```diff
@@ -0,0 +1,574 @@
+commonfields:
+  id: Vectra_Detect
+  version: -1
+
+beta: true
+name: Vectra_Detect
+display: Vectra Detect (Beta)
+# For future use
+# description: This integration allows to create incidents based on Vectra Accounts/Detections/Hosts objects
+description: This integration allows to create incidents based on Vectra Detections/Hosts objects
+category: Network Security
+defaultclassifier: Vectra Detect
+defaultmapperin: Vectra Detect - Incoming Mapper
+fromversion: 6.5.0
```

Comment:
Hi @AradCarmi , because I've done all my development/testing with a 6.5.0 version.
So I though it would be more logic to set it as my "from" version.
Can I change it to 6.2.0 without any issue (I don't know if some helpers/commands are only available in the 6.5.0 and not in the 6.2.0) ?

Reply:
I understand why you did it but, it seems that all the content entities you added require a lower version than 6.5.0 so our conventions suggest changing it to 6.2.0.
No worries I will change it 😄